src/block-components/separator/style.scss

```diff
@@ -5,8 +5,8 @@
 		.stk-inner-blocks {
 			z-index: 5;
 		}
-		> *:not(.stk-video-background) { // Don't change the order of any video backgrounds, they stay at the back.
-			z-index: 2;
+		> *:not(.stk-video-background, .stk-inner-blocks) { // Don't change the order of any video backgrounds, they stay at the back.
+			z-index: -5;
```

Comment:
You cannot do `-5` here because this can possibly hide the content behind other content that we don't control. Won't `2` work?